rotkehlchen/data_import/importers/bitmex.py

```diff
@@ -90,26 +93,40 @@ def _consume_deposits_or_withdrawals(
         amount = deserialize_asset_amount_force_positive(csv_row['amount'])
         fee = deserialize_fee(csv_row['fee']) if csv_row['fee'] != 'null' else Fee(ZERO)
         transact_type = csv_row['transactType']
-        category = AssetMovementCategory.DEPOSIT if transact_type == 'Deposit' else AssetMovementCategory.WITHDRAWAL  # noqa: E501
+        event_type = HistoryEventType.DEPOSIT if transact_type == 'Deposit' else HistoryEventType.WITHDRAWAL  # noqa: E501
         amount = AssetAmount(satoshis_to_btc(amount))  # bitmex stores amounts in satoshis
         fee = Fee(satoshis_to_btc(fee))
         ts = deserialize_timestamp_from_date(
             date=csv_row['transactTime'],
             formatstr=timestamp_format,
             location='Bitmex Wallet History Import',
         )
-        return AssetMovement(
+        extra_data = {'memo': f'Imported from BitMEX CSV file. Transact Type: {transact_type}'}
```

Comment:
This `Imported from ...` string was being put in `link` and I thought `memo` was maybe a better description of what it is than `link` when putting it in extra_data - but as you say, it is a bit silly to have this long human readable string for this, and I don't really think its really needed at all. I've simply removed this from the importers where it was being used.

If we do need a way to show that events are from a csv it would probably be better to add something like `'from_csv': True` to the extra_data of all history events from the data importers, which could be done here: https://github.com/rotki/rotki/blob/develop/rotkehlchen/data_import/utils.py#L102. This can be added relatively easily if we do decide we need it.